skrf/network.py

```diff
@@ -2243,12 +2243,14 @@ def write(self, file: str = None, *args, **kwargs) -> None:
         \*args, \*\*kwargs :
             passed through to :func:`~skrf.io.general.write`
 
-        Notes
-        -----
+
+        Note
+        ----
```

Comment:
I think `Notes` is actually correct, so you should change it back (multiple occasions in this commit):
https://numpydoc.readthedocs.io/en/latest/format.html#notes

Reply:
Notes is indeed correct and produces a section "Notes", while "Note" produce a pretty CSS bloc of note

Reply:
I see, thanks! So it's both fine and just a matter of taste?

Reply:
I guess so. I like the CSS one, it's more stylish ;)